### PR TITLE
STP-3076: Dell management network topics

### PR DESCRIPTION
**Pull request by nrockershousen**
## Summary and Scope

No major content changes. These edits are aligned with what was done for the top-level topics and Aruba topics. Here is a general list of changes:
- Organization of content
- Several procedures are currently formatted as topics without steps
- Grammar and formatting
- Unclear steps/instructions

## Issues and Related PRs

* Resolves [STP-3076](https://jira-pro.its.hpecorp.net:8443/browse/STP-3076)
* Change will also be needed in `release/1.2`

## Testing

N/A

### Tested on:

N/A

### Test description:

N/A

## Risks and Mitigations

None.

### CODE REVIEW

**Comment by mharding-hpe** [comment]
This needs to be rebased. Also, note that it is now targeted for the main branch. This is appropriate -- just be sure to backport or cherry-pick it into all appropriate release branches.

**Review by lukebates123 — APPROVED** [review]

File: operations/network/management_network/dell/vlan_trunking_8021q.md [hunk 1]
```diff
@@ -1,38 +1,38 @@
-# VLAN trunking 802.1Q
+# VLAN Trunking 802.1Q
 
 A trunk port carries packets on one or more VLANs specified. Packet that ingress on a trunk port are in the VLAN specified in its 802.1Q header, or native VLAN if the packet has no 802.1Q header. A packet that egresses through a trunk port will have an 802.1Q header if it has a nonzero VLAN ID. Any packet that ingresses on a trunk port tagged with a VLAN that the port does not trunk is dropped.
```
Comment (mharding-hpe) [inline operations/network/management_network/dell/vlan_trunking_8021q.md:3]:
```suggestion
A trunk port carries packets on one or more specified VLANs. Packet that ingress on a trunk port are in the VLAN specified in its 802.1Q header, or native VLAN if the packet has no 802.1Q header. A packet that egresses through a trunk port will have an 802.1Q header if it has a nonzero VLAN ID. Any packet that ingresses on a trunk port tagged with a VLAN that the port does not trunk is dropped.
```

File: operations/network/management_network/dell/vlan_trunking_8021q.md [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
+## Expected Results
 
-* Step 1: You can create and enable multiple VLAN interfaces
-* Step 2: You can assign the trunk VLAN interfaces
+1. Administrators can create and enable multiple VLAN interfaces
+2. Administrators can assign the trunk VLAN interfaces
 
 
-[Back to Index](../index.md)
+[Back to Index](index.md)
```
Comment (mharding-hpe) [inline operations/network/management_network/dell/vlan_trunking_8021q.md:38]:
```suggestion
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

**Comment by nrockershousen** [comment]
The backport for this ticket was merged prematurely. Opening up a new PR for release/1.2.

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**